src/octod/api/releases.d

```diff
@@ -24,11 +24,12 @@ import octod.core;
         tag  = tag used for the release
         title = title for the release
         content = content of the release
+        prerelease = true if this is a prerelease, else false
 
 *******************************************************************************/
 
 public void createRelease ( ref HTTPConnection connection, ref Repository repo,
-                            string tag, string title, string content )
+    string tag, string title, string content, bool prerelease = false )
```

Comment:
https://dlang.org/library/std/typecons/flag.html makes such API usage much more readable :P

Reply:
Fair point

Reply:
While I approve of this, the whole feature feels like it's hacking around around the lack of language support for just that